### PR TITLE
Sana Ullah: [Fixed Borrowed POST and PUT endpoints]

### PR DESCRIPTION
**Pull request by Sana-Ullah786**
This PR fixes the Endpoints of:

- **Borrowed POST** with respect to new status Table.
- **Borrowed PUT (return borrowed) ** with respect to new Status Table.
**Please Check it Critically and Leave Comments.**
**Ticket Link:**
https://app.asana.com/0/1204411152293714/1204642324377428/f

### CODE REVIEW

File: tests/test_borrowed_put.py [hunk 1]
```diff
@@ -50,160 +50,160 @@ def create_borrowed_entry_in_db(test_db: sessionmaker, is_librarian: bool) -> in
...
-            headers={"Authorization": f"Bearer {token}"},
-        )
-        logging.info(
-            " Response status code with Token = "
-            + token
-            + " "
-            + str(response.status_code)
-        )
-        assert response.status_code == status.HTTP_401_UNAUTHORIZED
+# # Test case for create borrowed (PUT /borrowed/{borrowed_id})
```
Comment (mtalhatahir2001) [inline tests/test_borrowed_put.py:53]: Kindly remove the commented code if not required.

File: tests/test_borrowed_put.py [hunk 2]
```diff
@@ -50,160 +50,160 @@ def create_borrowed_entry_in_db(test_db: sessionmaker, is_librarian: bool) -> in
...
-            headers={"Authorization": f"Bearer {token}"},
-        )
-        logging.info(
-            " Response status code with Token = "
-            + token
-            + " "
-            + str(response.status_code)
-        )
-        assert response.status_code == status.HTTP_404_NOT_FOUND
+# # Test case for create borrowed (PUT /borrowed/{borrowed_id})
```
Comment (mtalhatahir2001) [inline tests/test_borrowed_put.py:89]: Same as mentioned above remove the commented code if not required.

**Review by mtalhatahir2001 — CHANGES_REQUESTED** [review]

File: tests/test_borrowed_put.py (hunk 2 above)
Comment (Sana-Ullah786) [reply tests/test_borrowed_put.py]: Fixed.

File: tests/test_borrowed_put.py (hunk 1 above)
Comment (Sana-Ullah786) [reply tests/test_borrowed_put.py]: Fixed.

**Review by mustansirMuz — APPROVED** [review]
LGTM

**Pull request merged**